src/patternfly/components/Card/card.scss

```diff
@@ -3,9 +3,12 @@
   --pf-c-card--BackgroundColor: var(--pf-global--BackgroundColor--100);
   --pf-c-card--BoxShadow: var(--pf-global--BoxShadow--sm);
   --pf-c-card--m-hoverable--hover--BoxShadow: var(--pf-global--BoxShadow--lg);
-  --pf-c-card--m-selectable--hover--BoxShadow: var(--pf-global--BoxShadow--lg);
+  --pf-c-card--m-selectable--m-selected--BoxShadow: var(--pf-global--BoxShadow--lg);
   --pf-c-card--m-selectable--m-selected--before--Height: var(--pf-global--BorderWidth--lg);
   --pf-c-card--m-selectable--m-selected--before--BackgroundColor: var(--pf-global--active-color--100);
+  --pf-c-card--m-selectable--hover--BoxShadow: var(--pf-c-card--m-selectable--m-selected--BoxShadow);
+  --pf-c-card--m-selectable--focus--BoxShadow: var(--pf-c-card--m-selectable--m-selected--BoxShadow);
+  --pf-c-card--m-selectable--active--BoxShadow: var(--pf-c-card--m-selectable--m-selected--BoxShadow);
```

Comment:
The downside to sharing the vars like this is that if we need to update `--m-selected--BoxShadow` to be unique from the others, that will be a breaking change. The upside to this is if someone wants to change the box shadow for all of these values, they only have to change 1 thing. Typically we don't share the vars like this though, except in pretty specific instances (like a spacer to accommodate the value of a component's border width or something like that). The button is a good example. I would be in favor of making the value for these `var(--pf-global--BoxShadow--lg);`

Reply:
updated